geps/gep-1897/index.md

```diff
@@ -149,61 +117,53 @@ certificate, we implement a metaresource named `BackendTLSPolicy`, that was prev
-to Gateway cert validation, but as mentioned, BackendTLSPolicy is used primarily to signal what the application
-developer expects in the connection.  Granting this expectation to any other role would blur the lines between role
-responsibilities, which compromises the role-oriented design principle of Gateway API. As mentioned in Non-goal #8,
-providing a mechanism for the cluster operator gateway role to override gateway to backend TLS settings is not covered
-by this proposal, but should be addressed in a future update.  One idea is to use two types: ApplicationBackendTLSPolicy,
-and GatewayBackendTLSPolicy, where the application developer is responsible for the former, the cluster operator is
-responsible for the latter, and the cluster operator may configure whether certain settings may be overridden by
-application developers.
-
-The BackendTLSPolicy must contain these configuration items to allow the Gateway to operate successfully
+A new [metaresource](../gep-713/index.md), `BackendTLSPolicy`, will be introduced.
+
+With TLS, there are, of course, two components involved: the client (the gateway implementation) and the server (an application, likely running in the cluster).
+The Gateway API cannot control the server behavior.
+This is a choice the application developer makes in their code, whether or not (and how) they serve TLS.
+The application developer may not even be aware of the Gateway API at all.
+
+Therefor, the goal of this API is to tell the client *whether or not* and *how* to originate TLS to the backend.
+
+However, in many common cases these roles are actually inverted.
+While it is ultimately the client that must decide how to connect, commonly they are unaware of the details of the destination.
+In these situations, an application developer may wish to actually tell the clients how *they* should connect.
+
+Fortunately, these two scenarios can co-exist.
+`BackendTLSPolicy` will allow an application developer to present a *default* for clients that have not configured their own, but also allow clients to explicitly configure their own settings.
+
+This serves a variety of use cases:
+* A service producer may be unaware or unwilling to provide a `BackendTLSPolicy` for clients, despite serving TLS. In this case, a client can provide their own settings to connect
+  * Alternatively, a service producer may be unsure of how clients are connecting to them. `BackendTLSPolicy` implies *originating* TLS from the implementation, but the implementation could already be handling passthrough TLS.
+* A service producer may provide TLS settings that are unacceptable to the client. For example, an `insecureSkipVerify` option. The client can override this with their own more secure settings.
+* A service may be connected from a variety of context, some requiring TLS origination and others not. Allowing client-driven control allows both of these to co-exist.
```

Comment:
These should be able to co-exist already using two or more targetRefs on the policy.  These are type [LocalPolicTargetReferenceWithSectionName](https://github.com/kubernetes-sigs/gateway-api/blob/main/apis/v1alpha2/policy_types.go#L84-L101 ) with sectionName corresponding to port name for a Service.

Reply:
I believe this comment has been addressed by the recent changes that are now utilizing targetRefs. Please LMK if I misunderstood